srcpkgs/linux-steam-integration/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'linux-steam-integration'
+pkgname=linux-steam-integration
+version=0.7.3
+revision=1
+archs="x86_64 i686"
```

Comment:
If it works with flatpak steam, we should remove the limitation and allow `x86_64* i686*`.

Reply:
I'm pretty sure this does nothing for flatpak. (except ofc if you install it inside of flatpak).